index.js

```diff
@@ -126,11 +132,30 @@ function validateStringOrThrow(name, value, regexp){
 	}
 }
 
+// any -ve num is coerced to -1
+
+function constructMaxTokensString( num, maxLength ) {
+	if (maxLength < 2) {
+		throw new Error( "constructMaxTokensString: maxLength(", maxLength, ") is too short to accommodate a possible value of num=-1" );
```

Comment:
I think you have got the arity and use of `Error` confused with that of `console.log`.

The Error constructor function has an arity of 3, all the parameters are optional - `new Error(message, fileName, lineNumber)`

The `console.log` method has an arity of n, all parameters are optional.

To resolve the issue you can concatenate the strings together using `+` operator - `"a" + "b"`. Another solution would be to replace the String literals with a single Template literal `` `a ${2+2} ` ``.